src/compat/libc/stdlib/wcstombs.c

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file
+ * @brief wcstombs function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
```

Comment:
These two headers (`size_t.h`, `wchar_t.h`) are included in `stdlib.h`, so you dont need to include them.